src/main/java/sirius/db/mixing/properties/BaseEntityRefProperty.java

```diff
@@ -180,6 +192,20 @@ public void link() {
                                 + "The only supported DeleteHandler is IGNORE!.", this, getDescriptor());
                 return;
             }
+
+            // If a cascade delete handler is present and the referenced entity is not explicitely marked as
+            // "non complex" and we're within the IDE or running as a test, we force the system to compute / lookup
+            // the associated NLS keys which might be required to generated appropriate deletion logs or rejection
+            // errors. (Otherwise this might be missed while developing or testing the system..)
+            if (getReferencedDescriptor().getAnnotation(ComplexDelete.class).map(ComplexDelete::value).orElse(true)
```

Comment:
is this the right default, shouldnt this be `orElse(false)`?

Reply:
s.a.